index.js

```diff
@@ -37,14 +37,13 @@ const command = parsedArgs._[0];
 
 if (command === 'list-commands') {
   require('./commands/list-commands');
-  process.exit(0);
+} else {
+  exec(command)
+    .catch((err) => {
+      if (err && err.code === 'ENOENT') {
+        console.error(`Error: "${command}" is not an available ${cmd} command\n`);
+        showHelp();
+      }
+      process.exit(1);
```

Comment:
the node docs recommend not using this method of exiting, but they say use  `process.exitCode = 1` rather than `process.exit(1)` so the node engine can shut down gracefully

see https://nodejs.org/api/process.html#process_process_exitcode and the entry below it - https://nodejs.org/api/process.html#process_process_exitcode